internal/daemon/controller/handlers/scopes/scope_service.go

```diff
@@ -573,6 +573,13 @@ func sortKeys(keys []*pb.Key) {
 	sort.SliceStable(keys, func(i, j int) bool {
 		return keys[i].GetId() < keys[j].GetId()
 	})
+	// we also want to sort the key versions by version id so they are in
+	// descending order (newest first)
```

Comment:
Nit: If we want to sort by newest first, do we not want the sort to be on the `CreateTime` 😛. Should this say "sort based on highest version"?

Reply:
what we really want here is "sort based on order in which the version(s) were in use for encryption" but that seems kind of verbose and I think the point come across by just saying newest first